ci/datasources/regions.yaml

```diff
@@ -0,0 +1,29 @@
+regions:
```

Comment:
Correct, right now this approach wouldn't work for GovCloud either, since we are setting those regions here.

An ideal solution for another PR would be: in the generator pipeline, list regions and generate this file!